shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -542,6 +542,11 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
           [[[NSMutableArray alloc] init] autorelease];
       for (int32_t action_id : node.customAccessibilityActions) {
         blink::CustomAccessibilityAction& action = actions_[action_id];
+        if (action.overrideId != -1) {
+          // iOS does not support overriding standard actions, so we ignore any
```

Comment:
I believe back in the days for iOS we wanted to generate a custom hint based on the overridden actions if no hint had been provided. This could happen on the framework side though.